base_partner_sequence/tests/test_base_partner_sequence.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
```

Comment:
@elicoidal no guidelines for headers in https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md